index.html

```diff
@@ -1,185 +1,221 @@
+                                                                         // Add error to incorrect words
+                                                                         this.correctedWords.forEach(word => {
+                                                                                 const existingCorrection = existingCorrections.find(exWord => exword.text === word.text)
+                                                                                 console.log(word);
+                                                                                 newTex = newTex.replace(word.text, `<span class="text-error">${word.text}</span>`)
+                                                        
+                                                                         });
+                                                                         this.richText = newTex
+                                                                         this.event.target.blur();//disable cursor
+                                                                })
+                                                                .catch(error => {
+                                                                        this.answer = 'Error! Could not reach the API. ' + error
+                                                                }).finally( ()=>{
+                                                                          this.loading = false;
+                                                                })
+                                                },
+                                                replaceWord(word) {
+                                                        console.log(this.richText);
+                                                        // replace word 
+                                                        this.richText = this.richText.replaceAll(`<span class="text-error">${word.text}</span>`, word.correct)
+                                                        // remove word from corrected words
+                                                        this.correctedWords = this.correctedWords.filter(correctedWord => correctedWord.correct !== word.correct)
+                                                  
+                                                        console.log(this.richText);
+                                                },
+                                                onInput(event) {
+                                                        const value = event.target.innerText;
+                                                        this.event = event
+                                                        this.text = value;
+                                                },
+                                        },
+                                        watch: {}  
+                                })
+                                // download function

```

Comment:
Great one Harry! This works for our download button. With this users can download their file from our software!